src/components/filter/row/TogglerRow.tsx

```diff
@@ -37,9 +37,9 @@ const TogglerRow = ({ config }: { config: FilterRowTogglerConfig }) => {
 						toggleValue();
 					}
 				}}>
-				<p className='toggler__label'>{firstLabel}</p>
+				<p className={`toggler__label ${value ? 'selected' : ''}`}>{firstLabel}</p>
```

Comment:
maybe you can apply class to `togglerClassName` and then use selectors to target first or last p tag?